source/Menu.cpp

```diff
@@ -220,6 +221,7 @@ namespace GTA
 			EntryTuple->Item2->Color = TextColor;
 			i++;
 		}
+		this->SelectedIndexChanged(this, gcnew SelectedIndexChangedArgs(this->mSelectedIndex));
```

Comment:
This is normal. In c++/cli ```event``` has a default value that does nothing, so you don't have to check if it's null (and you should get error if you try to do something like ```if (this->SomeEvent != nullptr)```).